l10n_th_partner/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Kitti Upariphutthiphong. <kittiu@gmail.com> (http://ecosoft.co.th)
```

Comment:
You can add yourself as contributor here.
No need to chagnge in README.rst, ocabot will take care.